helper/dockertest/dockerjobs/benchmark_run.go

```diff
@@ -6,23 +6,37 @@ package dockerjobs
 import (
 	"context"
 	"fmt"
+	"io"
+	"os"
 	"testing"
 
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
 	dockhelper "github.com/hashicorp/vault/sdk/helper/docker"
 )
 
 func CreateVaultBenchmarkContainer(t *testing.T, vaultAddr string, vaultToken string, configFile string) (func(), int64) {
 	ctx := context.Background()
+	binary := os.Getenv("BENCHMARK_BINARY")
+
+	if binary == "" {
+		t.Skip("only running docker test when $BENCHMARK_BINARY present")
+	}
+
+	api, err := dockhelper.NewDockerAPI()
+	imageRepo := "hashicorp/vault-benchmark"
+	imageTag := "latest"
+	tag, err := setupBenchmarkImage(ctx, imageRepo, imageTag, binary, api)
```

Comment:
It looks like we're missing some error checks in here. What happens if one of these calls returns an error?